constants/questions.json

```diff
@@ -67,5 +69,15 @@
     "question": "You are an animal lover",
     "type": "RESULT",
     "answers": []
+  },
+  {
+    "id": 6,
+    "question": "You should love animals.",
+    "type": "CONTINUE",
+    "answers": [
+      {
+        "route": 3
```

Comment:
The question with id: 6, does it route back to the question with id: 3? Or are routes and questions connected in some other way?

Reply:
Yes, it routes back to the question with id: 3. You can view the flowchart representing this question setup [here](https://lucid.app/lucidchart/6e4eaa1f-8f11-44d7-b3e9-a1a7d8e2e369/edit?). If that link doesn't work, go through the Notion: EdLaw > Spring 2022 > External Links: Flowcharts > Animal Form.